interface/eRxSOAP.php

```diff
@@ -31,6 +31,21 @@ class eRxSOAP
     private $soapSettings = array();
     private $siteId;
 
+    protected static function fixHtmlEntities($array, $xmltoarray)
```

Comment:
Won't you need to pass $array by reference here?

Reply:
also, fyi, shooting for 7.0.2 release in several weeks (rather than another 7.0.1 patch)

Reply:
Yes, thanks!

Reply:
Testing a JSON solution...